server/fly.io/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM rust:latest as builder
+
+# Make a fake Rust app to keep a cached layer of compiled crates
```

Comment:
Perhaps we can use `cargo-chef` for caching here?

Reply:
Probably no point. The dockerfile in the example repo already does a good enough job of the multi layer build cache dance, so deploying only takes a few seconds.